src/views/locale/LocaleDetails.tsx

```diff
@@ -78,11 +79,11 @@ const LocaleDefinitionSection: React.FC<{ locale: LocaleData }> = ({ locale }) =
         </div>
       )}
       {variantTag && (
-        <div>
-          <label>Variant:</label>
-          {variantTag} <em>[variant not in database]</em>
-        </div>
-      )}
+  <div style={{ marginTop: '1em' }}>
+    <h3>Variant</h3>
+    <VariantTagCard data={variantTag} />
```

Comment:
There isn't enough space for a card here -- but if you add a HoverableObjectName you will see the VariantCardTag on hover.
```suggestion
    <HoverableObjectName object={variantTag} />
```

In order for this to work you just need to update HoverableObject to handle variant cards, here: https://github.com/Translation-Commons/lang-nav/blob/master/src/views/common/HoverableObject.tsx#L24-L38